openpathsampling/snapshot_modifier.py

```diff
@@ -37,6 +38,7 @@ class SnapshotModifier(StorableNamedObject):
     __metaclass__ = abc.ABCMeta
 
     def __init__(self, subset_mask=None):
+        super(SnapshotModifier, self).__init__()
```

Comment:
Actually I think this was missing. It is necessary for storable objects. In rare cases you might
not do it. Just to point @dwhswenson to a potential mistake.